src/Pim/Bundle/CatalogBundle/Doctrine/Common/Saver/ProductSaver.php

```diff
@@ -21,27 +23,27 @@
  */
 class ProductSaver implements SaverInterface, BulkSaverInterface
 {
-    /** @var ObjectManager */
```

Comment:
At that time, it was the only interface available (`EntityManagerInterface` was not there). And we do not use anything else than ORM. Plus, I am now using a method in the ProductSaver that does not exists in the ObjectManager interface.